### PR TITLE
DOCSP-18456: adding a change stream page

### PR DESCRIPTION
**Pull request by norareidy**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-golang/blob/master/REVIEWING.md)

JIRA - <https://jira.mongodb.org/browse/DOCSP-18456>
Staging - <https://docs-mongodbcom-staging.corp.mongodb.com/golang/docsworker-xlarge/DOCSP-18456-change-stream-page/fundamentals/crud/read-operations/watch/>

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?

### CODE REVIEW

File: source/fundamentals/crud/read-operations/watch.txt [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+   {1658847091 38} documentKey:map[_id:ObjectID("62dfff538bc04f3816b8bcd1")] 
+   fullDocument:map[_id:ObjectID("62dfff538bc04f3816b8bcd1") 
+   rating:4 type:White Peony] ns:map[coll:ratings db:tea] operationType:insert]
+
+.. note::
+
+   The ``_data``, ``clusterTime``, and ``ObjectID`` values in the output will 
+   vary with each user.
+
+Pipeline Parameter
```
Comment (mongoKart) [inline source/fundamentals/crud/read-operations/watch.txt:96]: Issue: A reader who isn't familiar with change streams won't know what the pipeline parameter represents.

Suggestion: Rename to "Modify the Change Stream Output"

File: source/fundamentals/crud/read-operations/watch.txt [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+~~~~~~~~~~~~~~~~~~
+
+Use a pipeline parameter to modify the change stream output. This parameter must be 
+a ``non-nil`` value, but you can pass an empty aggregation pipeline to return an unfiltered
+change stream. Format the pipeline parameter as an array of documents, with each
```
Comment (mongoKart) [inline source/fundamentals/crud/read-operations/watch.txt:101]:
```suggestion
Use the pipeline parameter to modify the change stream output. Format the pipeline parameter as an array of documents, with each 
```

File: source/fundamentals/crud/read-operations/watch.txt [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+- ``$replaceRoot``
+- ``$replaceWith``
+- ``$redact``
+- ``$set``
+- ``$unset``
+
+Example
+```````
+
+The following example opens a change stream on the ``tea`` database but only watches for
```
Comment (mongoKart) [inline source/fundamentals/crud/read-operations/watch.txt:118]:
```suggestion
The following example opens a change stream on the ``tea`` database, but only watches for
```

File: source/fundamentals/crud/read-operations/watch.txt [hunk 4]
```diff
@@ -0,0 +1,235 @@
...
+      panic(err)
+   }
+   defer changeStream.Close(context.TODO())
+
+   for changeStream.Next(context.TODO()) {
+      fmt.Println(changeStream.Current)
+   }
+
+If you delete the ``tea.ratings`` document with a ``name`` value of 
+"``White Peony``" in a separate shell, this code will output the following:
```
Comment (mongoKart) [inline source/fundamentals/crud/read-operations/watch.txt:137]:
```suggestion
``"White Peony"`` in a separate shell, this code will output the following:
```

File: source/fundamentals/crud/read-operations/watch.txt [hunk 5]
```diff
@@ -0,0 +1,235 @@
...
+   if err != nil {
+      panic(err)
+   }
+   defer changeStream.Close(context.TODO())
+
+   for changeStream.Next(context.TODO()) {
+      fmt.Println(changeStream.Current)
+   }
+
+If you delete the ``tea.ratings`` document with a ``name`` value of
```
Comment (mongoKart) [inline source/fundamentals/crud/read-operations/watch.txt:136]:
```suggestion
If you delete the ``tea.ratings`` document with a ``type`` value of 
```

File: source/fundamentals/crud/read-operations/watch.txt [hunk 6]
```diff
@@ -0,0 +1,235 @@
...
+   "delete","clusterTime": {"$timestamp":{"t":"1658848539","i":"16"}},"ns": 
+   {"db": "tea","coll": "ratings"},"documentKey": {"_id": {"$oid":"62dfff538bc04f3816b8bcd1"}}}
+
+.. note::
+
+   The ``Watch()`` method was called on the ``tea`` database, so the code outputs
+   new delete operations in any ``tea`` collection.
+
+
+Opts Parameter
```
Comment (mongoKart) [inline source/fundamentals/crud/read-operations/watch.txt:153]: Issue: Change to a task to be consistent with previous headings

Suggestion: "Modify the Behavior of the Watch() Method"

File: source/fundamentals/crud/read-operations/watch.txt [hunk 7]
```diff
@@ -0,0 +1,235 @@
...
+~~~~~~~~~~~~~~
+
+You can use an opts parameter to modify the behavior of the ``Watch()`` method. This
+parameter is optional and can remain ``nil``.
```
Comment (mongoKart) [inline source/fundamentals/crud/read-operations/watch.txt:157]: Suggestion: "Use the opts parameter to modify the behavior of the ``Watch()`` method."

File: source/fundamentals/crud/read-operations/watch.txt [hunk 8]
```diff
@@ -0,0 +1,235 @@
...
+	if err != nil {
+		panic(err)
+	}
+	defer changeStream.Close(context.TODO())
+
+	for changeStream.Next(context.TODO()) {
+		fmt.Println(changeStream.Current)
+	}
+
+If you update the ``rating`` value of the "``Masala``" tea from ``10`` to ``9``,
```
Comment (mongoKart) [inline source/fundamentals/crud/read-operations/watch.txt:193]:
```suggestion
If you update the ``rating`` value of the ``"Masala"`` tea from ``10`` to ``9``, 
```

File: source/fundamentals/crud/read-operations/watch.txt [hunk 9]
```diff
@@ -0,0 +1,235 @@
...
+
+   {"_id": {"_data": "8262E01219000000062B022C0100296E5A1004FB2444E03F9A4C5AB3C7E
+   868E387116846645F6964006462DFFF3C5E043FA89EE1C04C0004"},"operationType": 
+   "update","clusterTime": {"$timestamp":{"t":"1658851865","i":"6"}},"fullDocument":
+   {"_id": {"$oid":"62dfff3c5e043fa89ee1c04c"},"type": "Masala","rating": {"$numberInt":"9"}},
+   "ns": {"db": "tea","coll": "ratings"},"documentKey": {"_id": {"$oid":"62dfff3c5e043fa89ee1
+   c04c"}},"updateDescription": {"updatedFields": {"rating": {"$numberInt":"9"}},"removedFields":
+   [],"truncatedArrays": []}}
+
+Without specifying the ``FullDocument`` option, the same update operation outputs the following:
```
Comment (mongoKart) [inline source/fundamentals/crud/read-operations/watch.txt:207]: Suggestion: "If you don't specify the ``FullDocument`` option, the same update operation no longer outputs the ``fullDocument`` value."

File: source/fundamentals/crud/read-operations/watch.txt [hunk 10]
```diff
@@ -0,0 +1,235 @@
...
+   :copyable: false
+
+   {"_id": {"_data": "8262E011D9000000242B022C0100296E5A1004FB2444E03F9A4C5AB3C7E86
+   8E387116846645F6964006462DFFF3C5E043FA89EE1C04C0004"},"operationType": "update",
+   "clusterTime": {"$timestamp":{"t":"1658851801","i":"36"}},"ns": {"db": "tea",
+   "coll": "ratings"},"documentKey": {"_id": {"$oid":"62dfff3c5e043fa89ee1c04c"}},
+   "updateDescription": {"updatedFields": {"rating": {"$numberInt":"9"}},"removedFields":
+   [],"truncatedArrays": []}}
+
+The output no longer includes the ``"fullDocument"`` value.
```
Comment (mongoKart) [inline source/fundamentals/crud/read-operations/watch.txt:219]: Suggestion: Delete

**Review by mongoKart — CHANGES_REQUESTED** [review]
A lot of (possibly confusing) comments, so let's plan to go over this together (you, me, and Jordan). Some structural changes to help the flow and make it task-oriented.

**Review by jordan-smith721 — APPROVED** [review]
Great job on this! LGTM

**Pull request merged**